dissect/target/plugins/os/windows/tasks.py

```diff
@@ -453,7 +939,7 @@ def parse_task(self, entry):
         task.data = task.get_raw("Data")
 
         record = TaskRecord(
-            uri=uri.from_windows(task.uri) if task.uri else None,
+            path=path.from_windows(task.uri) if task.uri else None,
```

Comment:
Is this allowed? The XML element is `"RegistrationInfo/URI"`. If the contents are only paths or UNC paths, the `path` class can handle it, generic URI's however not.